templates/rhpam710-authoring.yaml

```diff
@@ -323,6 +323,11 @@ parameters:
   name: AUTH_LDAP_BIND_CREDENTIAL
   example: "Password"
   required: false
+- displayName: LDAP Flag Login Module
+  name: AUTH_LDAP_LOGIN_MODULE
+  description: "A flag to set login module to optional. The default value is required"
+  required: false
+  value: "optional"
```

Comment:
Shouldn't this `example` instead of `value`?

Reply:
Great catch @Kaitou786 . Yes there should be used example. Right now we are overriding default value `required`.
@msmagnanijr please fix this env var in all places

Reply:
@Kaitou786 good catch, tks!